packages/beeq/src/components/accordion/bq-accordion.tsx

```diff
@@ -232,7 +232,7 @@ export class BqAccordion {
             </slot>
           </div>
         </summary>
-        <div class="bq-accordion__body overflow-hidden" part="panel">
```

Comment:
We don't need `overflow-hidden` any longer ?

Reply:
We need ... my bad ... I'll remove all of these.